_posts/2017-11-14-Tiny-servant.markdown

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title:  "Tiny Servant"
+date:   2017-11-14
+source: Xanathar.169
+tags: [wizard, level3, transmutation]
+---
+
+**3rd-level Transmutation**
+
+**Casting Time**: 1 minute
+
+**Range**: Touch
+
+**Components**: V, S
+
+**Duration**: 8 hours
+
+You touch one Tiny, nonmagical object that isn't attached to another object or a surface and isn’t being carried by another creature. The target animates and sprouts little arms and legs, becoming a creature under your control until the spell ends or the creature drops to 0 hit points. See the stat block for its statistics.
+
+As a bonus action, you can mentally command the creature if it is within 120 feet of you. (If you control multiple creatures with this spell, you can command any or all of them at the same time, issuing the same command to each one.) You decide what action the creature will take and where it will move during its next turn or, you can issue a simple, general command, such as to fetch a key, stand watch, or stack some books, If you issue no commands, the servant does nothing other than defend itself against hostile creatures. Once given an order, the servant continues to follow that order until its task is complete.
```

Comment:
Misplaced comma: "during its next turn, or you can" not "during its next turn or, you can"